packages/core/src/test/testUtil.ts

```diff
@@ -634,3 +635,9 @@ export function getFetchStubWithResponse(response: Partial<Response>) {
 export function copyEnv(): NodeJS.ProcessEnv {
     return { ...process.env }
 }
+
+export function disableIfVscodeBelow(testContext: Mocha.Context, version: string) {
+    if (isMinVscode(version)) {
```

Comment:
This isn't really correct. `isMinVscode` checks if the current vscode is the min version.

The `version` arg is only relevant for when we bump the min version. It's unrelated to the check itself.

Reply:
`isMinVscode` might be less confusing if we change its argument to be dict-like:

    if (isMinVscode({ throwIfMinVscodeGte = version })

Reply:
Good catch, I was assuming this cutoff version could only be below the stable/insider versions, which isn't correct. I went ahead and avoided using `isMinVscode` altogether so that this function can be used more generally.